source/framework/core/inc/TRestAnalysisPlot.h

```diff
@@ -70,6 +70,7 @@ class TRestAnalysisPlot : public TRestMetadata {
         string title;
 
         Double_t normalize;
+        string scale;
```

Comment:
just a general reminder that we should not be using std namespace in header files

Reply:
Good catch, however I cannot see `using namespace std` in the header file, I guess it is inherited from other header files.

I will create an issue about this.